CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Consensus
+
+Since this config is used in code bases that affect various internal and external repos.
+It can potentially imply changes on repos with multi-million users so we need to be mindful
```

Comment:
The number of users here is over-dramatic. It affects a lot of developers in a lot of teams, but it does not directly affect the users. In many cases the code is even transpired after linting and it is mostly about making it easy to write code of high quality, with consistent syntax.